### PR TITLE
Add - Result Page

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Add - Result Page

Preview - https://deploy-preview-4--vishnu-quiz.netlify.app/pages/result.html

### CODE REVIEW

**Review by vrinda-mahajan — APPROVED** [review]
Looks good to merge.

File: pages/result.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+  <link rel="stylesheet" href="https://some-ui-components.netlify.app/export.css" />
+  <link rel="stylesheet" href="/stylesheets/result.css" />
+  <title>Result</title>
```
Comment (keshavkumar-in) [inline pages/result.html:11]: It's a good practice to put title before your links

**Review by keshavkumar-in — APPROVED** [review]
Nice work, just had a one suggestion, otherwise everything looks fine to me.

File: pages/result.html (hunk 1 above)
Comment (Vishnu-Aithal) [reply pages/result.html:11]: ok. Thanks For the Review

**Review by aryanandarnav13 — APPROVED** [review]
Good to go

**Pull request merged**